cmake/sasMacros.cmake

```diff
@@ -63,10 +66,9 @@ macro(generate_registry_code checker_paths)
    endforeach()
 endmacro(generate_registry_code)
 
-
-macro(add_sas_test_new)
+macro(add_sas_test)
```

Comment:
Same as for the `enable_sas`, it would be nice to have a comment above that explains the arguments.